application/src/main/resources/schedule.yml

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2020 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##
+## The schedules can be defined using exact keyspace/table names.
+## It's also possible to define the names using quoted regular expressions.
+##
+## When the repair scheduler configures repairs it adheres to the following order:
+## | Keyspace regex? | Table regex? |
```

Comment:
What about this:

When the repair scheduler configures repairs it configures in the following order:
1. Defined tables (kb1/tb1)
2. Defined keyspaces with regex tables (kb1/[a-Z]123)
3. Regex Keyspaces with defined tables ([a-Z]abc/tb2)
4. Regex keyspaces with regex tables ([a-Z]abc/[a-Z]123)

Reply:
Done